root/package.json

```diff
@@ -26,9 +26,27 @@
     "jstransformer-marked": "1.0.x",
     <% if (netlify == true) { %>"roots": "3.x",<%} -%>
     <% if (yaml == true) { %>"roots-yaml": "0.0.x",<%} -%>
-    <% if (grid == true) { %>"jeet": "6.1.x",<%} -%>
-    "rupture": "0.6.x",
-    "stylus": "0.52.x"
+    <% if (grid == true) { %>
+      "jeet": "6.1.x",
+      "rupture": "0.6.x",
+      "stylus": "0.52.x",
+      "lost": "6.6.x",
+    <%} -%>
+    <% for(var i = 0; i < postcssPLugins.length; i++) { %>
+      <% if (postcssPlugins[i].checked) { %>
+        <% if (postcssPLugins[i].name === "rucksack") { %>
+          , "rucksack-css": "0.8.x"
```

Comment:
Write a temp json file, then require it, then remove it with the after hook?

On Fri, Nov 13, 2015 at 6:03 PM, Henry Snopek <notifications@github.com>
wrote:

>> @@ -26,9 +26,27 @@
>>      "jstransformer-marked": "1.0.x",
>>      <% if (netlify == true) { %>"roots": "3.x",<%} -%>
>>      <% if (yaml == true) { %>"roots-yaml": "0.0.x",<%} -%>
>> -    <% if (grid == true) { %>"jeet": "6.1.x",<%} -%>
>> -    "rupture": "0.6.x",
>> -    "stylus": "0.52.x"
>> +    <% if (grid == true) { %>
>> +      "jeet": "6.1.x",
>> +      "rupture": "0.6.x",
>> +      "stylus": "0.52.x",
>> +      "lost": "6.6.x",
>> +    <%} -%>
>> +    <% for(var i = 0; i < postcssPLugins.length; i++) { %>
>> +      <% if (postcssPlugins[i].checked) { %>
>> +        <% if (postcssPLugins[i].name === "rucksack") { %>
>> +          , "rucksack-css": "0.8.x"
> @kylemac the prepended `,` doesn't work for json - any ideas on a better way to structure this?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/carrot/base/pull/14/files#r44844699